src/app.module.ts

```diff
@@ -1,9 +1,18 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { UsersModule } from './users/users.module';
+import { TypeOrmModule } from "@nestjs/typeorm";
+import { ConfigModule } from "@nestjs/config";
 
 @Module({
-  imports: [],
+  imports: [
+      // Domyślnie Nest nie czyta envów, trzeba zadeklarować poniżej, jakie pliki mają być użyte i w jakiej kolejności
```

Comment:
Code only in English, sry mate 😢 
btw. at least adding support for env files isn't too difficult xD